pkg/kubelet/network/exec/exec.go

```diff
@@ -29,6 +29,9 @@ limitations under the License.
 //         - setup, called after the infra container of a pod is
 //                created, but before other containers of the pod are created
 //         - teardown, called before the pod infra container is killed
+//         - status, called at regular intervals and is supposed to return a json
```

Comment:
Design: Why is this a periodic pull rather than something returned from SetUpPod() ?  Is there the expectation that a pod can change IPs after creation?  That might break a number of other components like the Service->Endpoint generation loop (not sure).